CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Models/GiniImageDocument.swift

```diff
@@ -44,6 +45,12 @@ final public class GiniImageDocument: NSObject, GiniCaptureDocument {
         self.previewImage = UIImage(data: data)
         self.isReviewable = true
         self.id = UUID().uuidString
+
+        switch imageSource {
+        case .appName(name: _) : isFromOtherApp = true
+        default: isFromOtherApp = false
+        }
```

Comment:
[Suggestion]: Could you please move `isFromOtherApp = false` to a new line to see what we are doing in a specific case?